pkg/kobject/kobject.go

```diff
@@ -47,22 +41,18 @@ var unsupportedKey = map[string]int{
 	"MemLimit":      0,
 	"MemSwapLimit":  0,
 	"NetworkMode":   0,
-	"Networks":      0,
```

Comment:
Thanks @kadel Yes it was added by a mistake. I added https://github.com/kubernetes-incubator/kompose/pull/230/commits/c29ef944cef35b2fe680e1a96e6c05acfc5114c4